step.yml

```diff
@@ -98,7 +98,9 @@ inputs:
       description: |-
         Use this input to specify custom options, to append to the end of the cordova-cli build command.
 
-        Cordova does not yet support the new build system made default in XCode 10 (https://github.com/apache/cordova-ios/issues/407).
+        Add `-- --packageType="bundle"` to build an Android bundle (aab). (https://github.com/apache/cordova-android/pull/764)
```

Comment:
there is a `--` typo

Reply:
not a typo: https://github.com/apache/cordova-android/pull/764#issuecomment-534076439

Reply:
sorry